exercises/simple-cipher/src/test/java/SimpleCipherStepThreeIncorrectKeyTest.java

```diff
@@ -0,0 +1,42 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class SimpleCipherStepThreeIncorrectKeyTest {
```

Comment:
Looks like these tests aren't part of the canonical data anymore so we can remove them 🙂